graph/common/Encoding.java

```diff
@@ -191,12 +194,12 @@ public boolean isBackward() {
                 pair(STRUCTURE_RULE.key, STRUCTURE_RULE)
         );
 
-
         private final byte key;
         private final PrefixType type;
         private final ByteArray bytes;
 
         Prefix(int key, PrefixType type) {
+            assert key < 200 : "Core prefix encoding should not use reserved range";
```

Comment:
Let's replace our usage of "Core" with "TypeDB" going forward. Also, we can make the error message more informative:

> The encoding range of >= 200 is reserved or TypeDB Cluster